src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -33,10 +33,10 @@ public class SwerveSubsystem extends SubsystemBase {
 
     public SwerveSubsystem() {
         // Initialize swerve modules
-        topLeftModule = new SwerveModule(tlDrive, tlSteer);
-        topRightModule = new SwerveModule(trDrive, trSteer);
-        bottomLeftModule = new SwerveModule(blDrive, blSteer);
-        bottomRightModule = new SwerveModule(brDrive, brSteer);
+        topLeftModule = new SwerveModule.TopLeft(tlDrive, tlSteer);
+        topRightModule = new SwerveModule.TopRight(trDrive, trSteer);
+        bottomLeftModule = new SwerveModule.BottomLeft(blDrive, blSteer);
+        bottomRightModule = new SwerveModule.BottomRight(brDrive, brSteer);
```

Comment:
add variables in Constants.java?

Reply:
Like the offsets? The CAN IDs are already in `Constants`.

Reply:
Yeah, the offsets, sorry